multiplayer/examples/sudoku/src/style/setGlobalStyle.ts

```diff
@@ -0,0 +1,60 @@
+const style = document.createElement("style")
```

Comment:
Why not do a .css file instead?

Reply:
Just a personal preference to handle everything in JS. Easier to include dynamic stuff if necessary (like in the game adapters, not needed here).